topics/admin/tutorials/data-library/slides.html

```diff
@@ -107,10 +107,7 @@
   * Allows authorized non-administrators to upload a directory of files.
   * Directory must contain sub-directories named the same as user's email.
   * Works well in combination with `ftp_upload_dir`.
+* `library_import_dir`
+  * Allows administrators to upload a directory of files.
 * `allow_path_paste`
   * Admin-only, allows importing from any path that the Galaxy's user has access to.
-
-???
-
-- If you use old library interface you can also set library_import_dir.
-- It specifies which folder admins may browse and import from.
```

Comment:
This has `video: true`, so speaker notes are mandatory. Is there something else to be said here about this slide?

Reply:
You're right, I've updated the speakers note in a rebase.